codec_specs.md

```diff
@@ -977,7 +981,8 @@ Block type identifier: 4
 Block type name: ITU T.35 metadata
 
 Description: the `BlockAdditional` data is interpreted as ITU T.35 metadata, as defined by ITU-T T.35
-terminal codes. `BlockAddIDValue` **MUST** be 4.
+terminal codes. `BlockAddIDValue` **MUST** be 4 as some WebM-oriented demuxers may ignore `BlockAddIDType`
```

Comment:
I don't know if this addition is a good idea. `BlockAddIDValue` **MUST** be 4, period. The reason why does not need to be included. It wasn't for `BlockAddIDType` 1.

Reply:
IMO it's OK to explain why there is this constraint.